rustls/src/compress.rs

```diff
@@ -228,3 +228,85 @@ mod feat_brotli {
 
 #[cfg(feature = "brotli")]
 pub use feat_brotli::{BrotliCompressor, BrotliDecompressor};
+
+#[cfg(all(test, any(feature = "brotli", feature = "zlib-rs")))]
+pub mod tests {
```

Comment:
The tests look good, but I think the ordering is inverted. I'd expect to see the `#[test]` fns up front and the helpers afterwards.

Reply:
sorry -- old habits die hard -- fixed.